src/main/java/com/flixmovies/imdb/dto/AllMoviesDto.java

```diff
@@ -0,0 +1,14 @@
+package com.flixmovies.imdb.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@AllArgsConstructor
+@Getter
+@Setter
+public class AllMoviesDto {
```

Comment:
Naming this class All Movies, it seems that the class contains all the movies. That's not the case.
Instead of naming the class by its usage, name it by its composition: MovieSummaryDto, MovieSmallDto, MovieCompactDto...